basics/architecture.md

```diff
@@ -1,106 +1,102 @@
-  actionable insights.
+  Understand how the components of Apache Pinot™ work together to create a scalable OLAP database that can delivery low-latency, high-concurrecy queries at scale.
 ---
 
 # Architecture
 
-This page introduces you to the guiding principles behind the design of Apache Pinot. Here you will learn the distributed systems architecture that allows Pinot to scale the performance of queries linearly based on the number of nodes in a cluster. You'll also learn about the two different types of tables used to ingest and query data in offline (batch) or real-time (stream) mode.
-
 {% hint style="info" %}
 We recommend that you read [Basic Concepts](concepts.md) to better understand the terms used in this guide.
 {% endhint %}
 
-## Guiding design principles
+Apache Pinot™ is a distributed OLAP database designed to serve real-time, user-facing use cases, which means handling large volumes of data with very low query latencies in the face of many concurrent queries. This imposes some unique and demanding requirements:
+
+* Ultra low-latency queries (as low as 10ms P95)
+* High query concurrency (as many as 100,000 queries per second)
+* High data freshness (streaming data available for query immediately upon ingestion)
+* Large data volume (up to petabytes)
 
-Engineers at LinkedIn and Uber designed Pinot to scale query performance based on the number of nodes in a cluster. As you add more nodes, query performance improves based on the expected query volume per second quota. To achieve horizontal scalability to an unbounded number of nodes and data storage, without performance degradation, we used the following principles:
+## Distributed design principles
 
-* **Highly available**: Pinot is built to serve low latency analytical queries for customer-facing applications. By design, there is no single point of failure in Pinot. The system continues to serve queries when a node goes down.
-* **Horizontally scalable**: Pinot scales by adding new nodes as a workload changes.
-* **Latency vs. storage:** Pinot is built to provide low latency even at high-throughput. Features such as segment assignment strategy, routing strategy, star-tree indexing were developed to achieve this.
-* **Immutable data**: Pinot assumes that all data stored is immutable. For GDPR compliance, we provide an add-on solution for purging data while maintaining performance guarantees.
-* **Dynamic configuration changes**: Operations such as adding new tables, expanding a cluster, ingesting data, modifying indexing config, and re-balancing must not impact query availability or performance.
+To accommodate large data volumes with stringent latency and concurrency requirements, Pinot is designed as a distributed database. As a distributed system, it has these goals:
+
+* **Highly available**: Pinot has no single point of failure. When operators choose data replication, the cluster can continue to serve queries when a node goes down.
```

Comment:
It refers to people. :)

Reply:
hmmm, so would this make sense:

- When a cluster is configured to replicate data, and a node goes down, the cluster is able to continue processing queries.

Reply:
Not quite. Replication is configured at the table level, so it would have to be:

> When tables are configured for replication, and a node goes down, the cluster is able to continue processing queries.

I was trying to avoid the passive voice, but this gets the mysterious operators out of the picture. :)